deploy/runtime/help/html/incentives.html

```diff
@@ -537,7 +537,7 @@ <h3 class="p_Heading3_atocs_" style="page-break-inside: avoid; page-break-after:
 <p class="p_VariableDescription"><span class="f_VariableDescription">The upper limit of the tax credit in dollars. For tax credits with no limits, type the value 1e+038.</span></p>
 <p class="p_VariableName" style="page-break-inside: avoid; page-break-after: avoid;"><span class="f_VariableName">Reduces Depreciation Basis</span></p>
 <p class="p_VariableDescription"><span class="f_VariableDescription">Available for non-residential projects when one of the depreciation options is active on the <a href="fin_residential.html" class="topiclink">Depreciation</a> page. </span></p>
-<p class="p_VariableDescription"><span class="f_VariableDescription">The check boxes determine whether the basis used to calculate federal depreciation, state depreciation, or both should be reduced by the tax credit amount. When you check the box for an ITC, SAM reduces the depreciable basis by 50% of the ITC amount as specified by U.S. IRS rules.</span></p>
+<p class="p_VariableDescription"><span class="f_VariableDescription">The check boxes determine whether the basis used to calculate federal depreciation, state depreciation, or both should be reduced by the tax credit amount. When you check the box for an ITC, SAM reduces the depreciable basis by 50% of the ITC amount as specified by U.S. Internal Revenue Service rules.</span></p>
 <h4 class="p_Heading4"><span class="f_Heading4">Specifying Annual ITC Values</span></h4>
```

Comment:
seems like this is correct for the patch and will need to be revised for the reserve accounts in the fall release